gateway/header.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 )
 
+const (
+	// XForwardedFor is a header to identify the origin of an IP address
+	XForwardedFor = "X-Forwarded-For"
+)
```

Comment:
What the reason to keep this constant here ?

Reply:
This is the key for the "X-Forwarded-For" value, which is the http request header. I figured the gateway package and header.go file would the most suitable place for this variable.

Could you suggest a better location for this variable?